yaptide/routes/simulation_routes.py

```diff
@@ -151,18 +146,20 @@ class _Schema(Schema):
 
     @staticmethod
     @requires_auth(is_refresh=False)
-    def post(user: UserModel):
+    def get(user: UserModel):
```

Comment:
Do I have to rewrite front end to send get instead of post?

Reply:
Yes, you have to change it.